src/lib/core/data-table/data-table.spec.ts

```diff
@@ -0,0 +1,255 @@
+import {async, ComponentFixture, TestBed} from '@angular/core/testing';
+import {Component, ViewChild} from '@angular/core';
+import {CdkTable} from './data-table';
+import {CollectionViewer, DataSource} from './data-source';
+import {CommonModule} from '@angular/common';
+import {Observable} from 'rxjs/Observable';
+import {BehaviorSubject} from 'rxjs/BehaviorSubject';
+import {customMatchers} from '../../matchers';
+import {CdkDataTableModule} from './index';
+
+describe('CdkTable', () => {
+  let fixture: ComponentFixture<SimpleCdkTableApp>;
+
+  let component: SimpleCdkTableApp;
+  let dataSource: FakeDataSource;
+  let table: CdkTable;
+  let tableElement: HTMLElement;
+
+  beforeEach(async(() => {
+    jasmine.addMatchers(customMatchers);
+    jasmine.addMatchers(tableCustomMatchers);
+
+    TestBed.configureTestingModule({
+      imports: [CommonModule, CdkDataTableModule],
```

Comment:
I think you can skip CommonModule here too since it's an import to CdkDataTableModule and nothing in the template for this test component that needs CommonModule.